rules/go_rules.build_defs

```diff
@@ -897,10 +898,23 @@ def go_get(name:str, get:str|list, repo:str='', deps:list=[], exported_deps:list
             licences = licences,
             module_major_version = module_major_version,
         )
-        outs += [getroot]
+        get_roots += [getroot]
```

Comment:
This probably needs a better name... This has all the locations where we have `thing/...` because in that case we might have `thing.a` AND a folder `thing` containing additional outs.